db/migrate/20200912021520_create_join_table_campaigns_fees.rb

```diff
@@ -0,0 +1,10 @@
+class CreateJoinTableCampaignsFees < ActiveRecord::Migration[6.0]
+  def change
+    create_join_table :campaigns, :fees do |t|
+      # t.index [:campaign_id, :fee_id]
+      # t.index [:fee_id, :campaign_id]
+    end
+
+    add_column :campaigns, :has_square_fee, :boolean, default: true
```

Comment:
Instead of making this a boolean, what if we just made it an optional ID reference to the Fee? If theres no Fee ID, that means there is no fee.